ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/aaa-accounting.j2

```diff
@@ -1,7 +1,20 @@
 {# eos - aaa accounting #}
 {% if aaa_accounting is arista.avd.defined %}
-{%     if aaa_accounting.exec.console.type is arista.avd.defined and aaa_accounting.exec.console.group is arista.avd.defined %}
-aaa accounting exec console {{ aaa_accounting.exec.console.type }} group {{ aaa_accounting.exec.console.group }}
+{%     if aaa_accounting.exec.console.type is arista.avd.defined %}
+{%         set aaa_accounting_exec_console_cli = "aaa accounting exec console" %}
+{%         if aaa_accounting.exec.console.type is arista.avd.defined('none') %}
+{%             set aaa_accounting_exec_console_cli = aaa_accounting_exec_console_cli ~ " none" %}
+{{ aaa_accounting_exec_console_cli }}
```

Comment:
```suggestion
```
You already print this in the end - outside the if/else block.
We should test this condition, since that would exhibit this issue.